etl/steps/data/garden/faostat/2022-05-17/faostat_fbsc.py

```diff
@@ -196,6 +188,11 @@ def run(dest_dir: str) -> None:
     # Create new dataset in garden.
     dataset_garden.save()
 
+    # Create a long table (with item code and element code as part of the index).
+    log.info("faostat_fbsc.prepare_long_table", shape=data.shape)
+    data_table_long = prepare_long_table(data=data)
+    _assert_df_size(data_table_long, 2000)
+
```

Comment:
I explicitly moved the use of `prepare_long_table` and `prepare_wide_table` to the "Process data" section, instead of the "Preparing outputs" section, since they are very heavy processing steps. But it's not a big deal, if you think this makes more sense.

Reply:
I moved it there because we can save the long data, then free it from the memory and continue with wide data. We could also move those code blocks into their own functions (something like `save_long_dataset` and `save_wide_dataset`).

Anyway, not a big deal for me either (and memory wise this only saves about 1GB), so I'm putting your code back.